### PR TITLE
Update Encoding information, add example

### PR DESCRIPTION
**Pull request by andreasscherbaum**

### CODE REVIEW

**Comment by andreasscherbaum** [comment]
Again missing the ID for the XML tag.

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml [hunk 1]
```diff
@@ -8,9 +8,13 @@
       with something else, such as a sequence of numbers or octets, to facilitate data stransmission
       and storage. Greenplum Database supports a variety of character sets, including
       single-byte character sets such as the ISO 8859 series and multiple-byte character sets such
-      as EUC (Extended UNIX Code), UTF-8, and Mule internal code. Clients can use all supported
+      as EUC (Extended UNIX Code), UTF-8, and Mule internal code. The server-side character set
+      is defined during database initialization, usually UTF-8 is used. Clients can use all supported
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: Isn't UTF-8 the default?
I would say UTF-8 is the default.

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml [hunk 2]
```diff
@@ -8,9 +8,13 @@
...
       character sets transparently, but a few are not supported for use within the server as a
-      server-side encoding.</p>
+      server-side encoding. When loading or inserting data into Greenplum Database, the DBMS will
+      transparently transform the data from the specified client encoding into the server encoding.
+      When sending data back to the client, the data is transformed again from the server character
+      set into the specified client encoding.</p>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: My proposed edit.

When loading or inserting data into Greenplum Database, Greenplum 
converts the data from the specified client encoding into the server encoding.
When sending data back to the client, Greenplum converts the data from the server 
character encoding into the specified client encoding.

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml [hunk 3]
```diff
@@ -19,4 +23,14 @@
...
+  <topic xml:lang="en">
+    <title>Changing the client-side character encoding</title>
+    <body>
+      <p>The client-side character encoding can be changed on the fly:</p>
+      <p>
+        <codeblock>SET client_encoding TO 'latin1';
+</codeblock>
+      </p>
+    </body>
+  </topic>
```
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: My proposed edits

 
\<topic id="topic103" xml:lang="en">
  \<title>Changing the Client-Side Character Encoding\</title>
  \<body>
    \<p>The client-side character encoding can be changed for a session by setting the
      server configuration paramter \<codeph>client_encoding\</codeph>\</p>
      \<codeblock>SET client_encoding TO 'latin1';</codeblock>
   \</body>
  \</topic>

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml (hunk 1 above)
Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: It is the default, but can be changed.

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml (hunk 3 above)
Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: Slightly changed, added the closing codeblock tag. Should the codeblock be in a p tag?

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml (hunk 2 above)
Comment (andreasscherbaum) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: Adapted.

File: gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml (hunk 1 above)
Comment (mkiyama) [inline gpdb-doc/dita/admin_guide/load/topics/g-character-encoding.xml]: How about
UTF-8 is the default and can be changed.

**Comment by andreasscherbaum** [comment]
Text updated

**Pull request merged**